imports/ui/components/music/playlists.js

```diff
@@ -0,0 +1,21 @@
+import './playlists.html';
+import { Template } from 'meteor/templating';
+import { Playlists } from '../../../api/playlists/playlists_collection.js';
+import { Shows } from '../../../api/shows/shows_collection.js';
+import { pagination } from 'meteor/kurounin:pagination';
+
+Template.musicPlaylists.onCreated(function() {
+  var self = this;
+  self.pagination = new Meteor.Pagination(Playlists, {sort: {showDate: -1}});
+  self.subscribe('allPlaylists');
```

Comment:
We should work on narrowing this down to a smaller number. After, say, a year or so of DJs entering playlists into the program, this could be quite a large chunk of data!